LO2ChannelStripComponent.py

```diff
@@ -193,7 +193,7 @@ def _lo2__on_solo_changed(self):
             self.send_default('/live/'+self._track_types[self._type]+'solo', self._track.solo)
 
     def _lo2__on_arm_changed(self):
-        if self._type == 0 and self._type is not None:
+        if self._type == 0 and self._type is not None and self._track.can_be_armed:
```

Comment:
When loading an existing Ableton Live set, LiveOSC2 was crashing. This was because it was trying to get the arm state on some tracks that don't have that property.